test/get/get_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/pkg/errors"
```

Comment:
Ahhhhh this is causing the CI failure. I think you want to import "errors" instead of "github.com/pkg/errors". Or personally, my preference is to import "fmt" and use `fmt.Errorf` instead.